src/components/network/network-table.js

```diff
@@ -1221,6 +1329,129 @@ const NetworkTable = (props) => {
         [rowFilter]
     );
 
+    const handleOpenPopupSelectColumnNames = () => {
+        setListColumnsNames(
```

Comment:
Isn't this just setting listColumnsName to the same value as the initial value ? It looks like both computation (here and the cols variable) do the same thing